v-next/hardhat/src/internal/builtin-plugins/network-manager/hook-handlers/hre.ts

```diff
@@ -48,5 +48,7 @@ async function createNetworkManager(
     context.artifacts,
     hre.userConfig,
     hre.config.chainDescriptors,
+    hre.globalOptions.config,
+    hre.config.paths.root,
```

Comment:
These two properties are required as input to the general `resolveUserConfigToHardhatConfig` function.
Let me know if there is a way of getting them from the existing passed in properties, I was unable to find them.